modules/serverless-rn-1-33-0.adoc

```diff
@@ -25,6 +25,10 @@
 
 * You can now use the Custom Metrics Autoscaler Operator to autoscale Knative Eventing sources for Apache Kafka sources, defined by a `KafkaSource` object. This functionality is available as a Technology Preview feature, offering enhanced scalability and efficiency for Kafka-based event sources within Knative Eventing.
 
+* You can now customize the internal Kafka topic properties when creating a Knative Broker with Kafka implementation. This improves efficiency and simplifies management.
+
+* The new trigger filters feature is now available as a Technology Preview. The new trigger filters are now enabled by default. It allows users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
```

Comment:
Could be fine as is, but seems repetitive to say "new trigger filters" twice.

I also combined the second and third sentences, just to get rid of the "it" - because the previous sentence uses "triggers" plural but then "it" is singular. It's fine to keep the sentences separate, but if you do, make sure the pronoun matches the plural triggers.

```suggestion
* The new trigger filters feature is now available as a Technology Preview. The trigger filters are enabled by default and allow users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
```

Reply:
Received LGTM from @mgencur @nainaz @jrangelramos